### ISSUE TITLE
AA Notification Forwarder crashing

### ISSUE THREAD

**Issue by NAmRoD21v** [ztNFny/AANotificationForwarder#9]
When I forward a notification with from my Homey app, AA Notification Forwarder crashes directly. The app does contain text as shown in the following screenshot. Also, the Homey app does send notifications without issues, which are visible on my phone. When I use another app, AA Notification Forwarder works as expected. 

![Screenshot_20240203-015448_Nova Launcher](https://github.com/ztNFny/AANotificationForwarder/assets/6766394/79f226bd-10c9-47c2-b6d5-634ba707a8bc)

When AA Notification Forwarder crashes I'll get the following error message:
```
FATAL EXCEPTION: main
Process: xda.xlafbk.aanotificationforwarder, PID: 8585
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.lang.Object.toString()' on a null object reference
    at xda.xlafbk.aanotificationforwarder.NotificationForwarder.onNotificationPosted(NotificationForwarder.java:116)
    at android.service.notification.NotificationListenerService.onNotificationPosted(NotificationListenerService.java:452)
    at android.service.notification.NotificationListenerService$MyHandler.handleMessage(NotificationListenerService.java:2336)
    at android.os.Handler.dispatchMessage(Handler.java:106)
    at android.os.Looper.loopOnce(Looper.java:201)
    at android.os.Looper.loop(Looper.java:288)
    at android.app.ActivityThread.main(ActivityThread.java:7924)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
```

**Comment by ztNFny** [comment]
The cause for the crash is that bundle.containsKey("android.bigText") is true but bundle.getCharSequence("android.bigText") is null. Very strange what they did there.
Will add some additional checks to avoid the crash, new release coming up today.

**Comment by ztNFny** [comment]
fixed in 1.3
